Tests/LinuxMain.stencil

```diff
@@ -0,0 +1,16 @@
+// swiftlint:disable:previous vertical_whitespace
+// swiftlint:disable trailing_comma
+@testable import MongoSwiftTests
+import XCTest
+
+{% for type in types.classes|based:"XCTestCase" %}
+{% if not type.annotations.disableTests and type.name != "MongoSwiftTestCase" %}extension {{ type.name }} {
+  static var allTests: [(String, ({{ type.name }}) -> () throws -> Void)] = [
+  {% for method in type.methods %}{% if method.parameters.count == 0 and method.shortName|hasPrefix:"test" %}  ("{{ method.shortName }}", {{ method.shortName }}),
+  {% endif %}{% endfor %}]
+}
+
+{% endif %}{% endfor %}
+XCTMain([
+{% for type in types.classes|based:"XCTestCase" %}{% if not type.annotations.disableTests and type.name != "MongoSwiftTestCase" %}  testCase({{ type.name }}.allTests),
+{% endif %}{% endfor %}])
```

Comment:
It would be easier to read this file if it was indented similarly to actual code. If it's left this way purposefully so that the code it generates is more readable, then that should trump the readability of this file.

Reply:
good point. I've reformatted the file a bit to make it more readable (some more line breaks etc.) while still preserving the desired output code formatting. also realized the template was using tabs with width 2 instead of 4 spaces for indenting so I fixed that 😱